runner/runner_test.go

```diff
@@ -250,15 +253,20 @@ func Test_runTracerouteMulti(t *testing.T) {
 	}
```

Comment:
The order of the errors is also indeterminate.  It usually is
"error running traceroute 1\nerror running traceroute 2"
but I have seen it as
"error running traceroute 2\nerror running traceroute 1".